src/VolleyManagement.Backend/VolleyManagement.Domain/TeamsAggregate/CreateTeamDto.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace VolleyManagement.Domain.TeamsAggregate
+{
+    public class CreateTeamDto
```

Comment:
what about Roster?

Reply:
CreateTeam.feature does have specific data to pass - there is no roster.